docs/proposals/kubelet-eviction.md

```diff
@@ -76,8 +76,10 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
 | nodefs.available   | nodefs.available := node.stats.fs.available |
 | nodefs.inodesFree | nodefs.inodesFree := node.fs.inodesFree |
+| nodefs.inodesPercentAvailable | nodefs.inodesPercentAvailable := node.fs.inodesFree / node.fs.inodes |
```

Comment:
> As a user, I worry that failure to omit a % could cause drastically different results that only manifest themselves when bad things are happening to their nodes. I said to evict when nodefs.inodesFree<10, oh wait, I meant 10%!

Don't we already have this situation with quantity in the kubernetes API? For example, it is very easy to mistype `100m` as `100M` or vice-versa.